crates/oxc_linter/src/config/rules.rs

```diff
@@ -155,6 +155,10 @@ fn transform_rule_and_plugin_name<'a>(
         return (rule_name, "jest");
     }
 
+    if plugin_name == "typescript" && is_eslint_rule_adapted_to_typescript(rule_name) {
+        return (rule_name, "eslint");
+    }
```

Comment:
question, so https://github.com/oxc-project/oxc/blob/6dd71c6d050630fe3ee29728f906884ab627987a/crates/oxc_linter/src/rules/typescript/no_unused_expressions.rs#L1

no-unused-expressions exists in `typescript` should it be moved to eslint?

Reply:
good catch. feel free to create an PR.

This should also fix the docs in #2180

Reply:
thanks

https://github.com/oxc-project/oxc/pull/7624